.gitlab/.gitlab-ci.yml

```diff
@@ -189,10 +190,7 @@ apps-halide:
     - git submodule update --init --recursive -- hardware/deps/*
     - make patch-hw
     - cd hardware
-    - |
```

Comment:
Should we add a matrix as well, as we do on GitHub, or is this redundant?

Reply:
I don't think it's necessary.